h/static/scripts/site.js

```diff
@@ -3,7 +3,9 @@
 // Configure error reporting
 const settings = require('./base/settings')(document);
 if (settings.raven) {
-  require('./base/raven').init(settings.raven);
+  const raven = require('./base/raven');
+  raven.init(settings.raven);
+  raven.setUserInfo({userid: settings.raven.userid});
```

Comment:
> I'm pretty sure you want to use the key id, not userid, as otherwise Sentry won't display it properly

You're right. Sentry will still display the userid on reports if the key "userid" is used, but won't treat it as a unique identifier for the user.